### PR TITLE
returns nil if API get response status is 400 or more

### PR DESCRIPTION
**Pull request by amithgeorge**
This PR needs to be merged, for error responses to `API.config` in rails app to not be cached - https://github.com/quintype/southlive/commit/e9a8a6265e17d3475727f43fa7dd384ffc06e9dc

### CODE REVIEW

File: lib/quintype/api.rb [hunk 1]
```diff
@@ -199,6 +199,8 @@ def _post_returning_headers(url_path, body, session_cookie=nil)
 
     def _get(url_path, *args)
       response = @@conn.get(@@api_base + url_path, *args)
+      return nil if response.status >= 400
```
Comment (ghost) [inline lib/quintype/api.rb:202]: `!= 200` is more to the point.

Comment (ghost) [inline lib/quintype/api.rb:202]: Also, put in a note saying a `break` will have to be involved.

Comment (sharangj) [inline lib/quintype/api.rb:202]: 👍

Comment (amithgeorge) [inline lib/quintype/api.rb:202]: I checked against 200 at first, but wasn't sure if the api wouldn't return a 302. I think there is some api somewhere in sketches that does return a  302.

**Pull request merged**